src/utils/database.ts

```diff
@@ -4407,7 +4570,12 @@ const createCryptoTransaction = async (
     confirmed_at: new Date().toISOString(),
     currency: Currency.USD,
     total_fee: feeInUSD,
-    metadata: {},
+    metadata: {
+      sender_address: account_address.toLowerCase(),
+      ...(transactionRequest.receiver_address && {
+        receiver_address: transactionRequest.receiver_address.toLowerCase(),
```

Comment:
The receiver address here shouldn't be client-dependent. What I expect you to do is modify "getTransactionFeeThirdweb" to get the receiver address and use that here.